src/preamble.js

```diff
@@ -1055,8 +1060,16 @@ function createWasm(env) {
 #endif
   }
 
+#if 'emscripten_generate_pc' in addedLibraryItems
+  addRunDependency('offset-converter');
```

Comment:
Is this code in another PR too? I thought I already wrote a comment on this.

Reply:
Yes, this code came from #8762.